wiki/BBCode/de.md

```diff
@@ -0,0 +1,304 @@
+# BBCode
+
+**BBCode** ist eine [Auszeichnungssprache](https://de.wikipedia.org/wiki/Auszeichnungssprache) (Markup), die im osu!-Forum und, in einem größeren Umfang, in den meisten Foren im Internet genutzt wird. Damit die Rich-Text-Formatierung möglich wird, umranden Tags Text, um die Formatierung, Attribute, Einbettung usw. festzulegen. Es wird auf verschiedenen Seiten auf der osu!-Webseite verwendet, wie z. B. Forum-Beiträge, Signaturen, Benutzerseiten und Beatmapbeschreibungen.
+
+![Der Forum-Beitragseditor mit seinen Buttons](img/editor-DE.jpg?1 "Die Box zum Editieren in den Foren")
+
+## Verhalten
+
+Wenn vor dem Drücken des Markup-Buttons kein Text markiert wurde, wird um den Cursor ein öffnendes sowie schließendes Tag im Beitragseditor hinzugefügt. Sofern Text vorher markiert wurde, werden die Tags um den markierten Text gesetzt.
+
+Benutzer, die die Formatierung in einem einzigen Textabschnitt setzen möchten, können BBCode-Tags um die Sektion setzen. Allerdings muss die Reihenfolge und die Verschachtelung der Tags **beachtet werden**. Ansonsten wird der Text nicht formatiert.
+
+Beispiele für korrekte und inkorrekte Benutzungen von verschachtelten Tags ist unten näher beschrieben:
+
+- `[centre][b]Text[/b][/centre]` ist korrekt
+- `[b][centre]Text[/b][/centre]` ist inkorrekt
+
+## Tags
+
+BBCode, wie viele andere Markup-Sprachen, formatiert Text durch ein System von Tags, die durch ein Paar eckiger Klammern (`[]`) gekennzeichnet sind. Diese Tags sind aufgeteilt in "öffnende" und "schließende" Tags, die durch das Einfügen eines Schrägstrichs (`/`) unterscheidbar werden. D. h., dass schließende Tags einen Schrägstrich direkt nach der öffnenden Klammer haben, wohingegen öffnende Tags dies nacht haben.
+
+Es ist auch wichtig zu wissen, dass öffnende Tags manchmal Gleichheitszeichen (`=`) enthalten, um URLs, Schriftgröße und andere derartige Elemente zu markieren.
+
+BBCode-Tags, die von der osu!-Webseite unterstützt werden, sind unten aufgelistet und näher erläutert.
+
+### Fettschrift
+
+```
+[b]Text[/b]
+```
+
+Das Tag `[b]` wird benutzt, um Text mit Hilfe von Fettschrift stark zu betonen. Die Schriftgröße wird hierbei nicht verändert.
+
+Schaltfläche in der Symbolleiste: ![Button für Fettschrift](img/bold.png "Fett")
+
+### Kursivschrift
+
+```
+[i]Text[/i]
+```
+
+Das Tag `[i]` wird benutzt, um Text mit Hilfe einer Schräglage leicht zu betonen (d. h. in kursiv setzen).
+
+Schaltfläche in der Symbolleiste: ![Button für Kursivschrfit](img/italic.png "Kursiv")
+
+### Unterstreichen
+
+```
+[u]Text[/u]
+```
+
+Das Tag `[u]` wird benutzt, um mit Hilfe einer horizontalen Linie Text hervorzuheben (d. h. unterstreichen). Die horizontale Linie wird von anderen Tags beeinflusst, wie Fettschrift und Kursivschrift.
+
+### Durchstreichen
+
+```
+[strike]Text[/strike]
+```
+
+*Tipp: "strikethrough" ist auch bekannt als "strike".*
```

Comment:
Does this note apply here? Sounds like it exclusively refers to the english title ("Strikethrough") .

Reply:
Yeah, right. Will remove it.